include/bluetooth/audio/capabilities.h

```diff
@@ -155,7 +155,7 @@ struct bt_audio_capability_ops {
 	 *
 	 *  @return 0 in case of success or negative value in case of error.
 	 */
-	int (*enable)(struct bt_audio_stream *stream, uint8_t meta_count,
+	int (*enable)(struct bt_audio_stream *stream, size_t meta_count,
 		      struct bt_codec_data *meta);
```

Comment:
Unrelated to this PR, but it seems this is inconsistent: in other places you have the array first and then the array size. Here it's the other way around. Any particular reason? Would be good to make this consistent throughout the codebase.

Reply:
Agreed, they should be the same. I suggest to follow the convention of the standard libraries that have the array before the size/length: https://github.com/zephyrproject-rtos/zephyr/discussions/41120#discussioncomment-2242601